Backend/Middleware/src/main/java/middleware/dto/JWTRequest.java

```diff
@@ -0,0 +1,15 @@
+package middleware.dto;
+
+import lombok.Data;
+
+/**
+ * Data Transfer Object (DTO) for representing a JWT request.
+ *
+ * <p>This class encapsulates a single JWT, which is used to identify 
+ * a user or an entity in the system. It provides getter and setter methods to 
```

Comment:
might have to innclude a hyperlink to project [lombok ](https://projectlombok.org/) cos the code doesnt have the typical boilerplate